datastore-cli/src/main/scala/org/allenai/datastore/cli/UploadApp.scala

```diff
@@ -42,20 +42,22 @@ object UploadApp extends App {
     help("help")
   }
 
-  parser.parse(args, Config()) foreach { config =>
-    val datastore = config.datastore match {
-      case Some(datastore) => datastore
-      case None =>
-        Common.printDefaultDatastoreWarning()
-        Datastore
+  Common.handleDatastoreExceptions {
+    parser.parse(args, Config()) foreach { config =>
+      val datastore = config.datastore match {
```

Comment:
I renamed `Config`, but I'm not very happy about that either. I picked up that naming scheme for scopt from someone else, @schmmd maybe, and this just means that now we have one more scopt naming scheme than we had before.